docs/v2/guides/gutenberg.md

```diff
@@ -11,112 +11,123 @@ Timber works with the Block Editor (also called Gutenberg) out of the box. If yo
 ### How to use ACF Blocks with Timber
 
-Before you can start using ACF Blocks, you must install the Advanced Custom Fields 5.8.0-beta version or later.
+Before you can start using ACF Blocks, you must install the Advanced Custom Fields 6.2.5 version or later.
 
-To create a content block, you first have to register it in **functions.php** or in a separate plugin:
+#### Add Blocks Directory Structure
 
-```php
-add_action('acf/init', 'my_acf_init');
+First, we will create a blocks directory in our theme file. This directory will house the individual block.json files that will provide the settings for each respective block.
+
+1. Create a **blocks** directory in the root of your theme: 
+2. Add a directory with the block name of your choice. Example: **my-block**
+3. Within your custom block directory, add the block.json file. 
+4. Also within your custom block directory, add a css file to reference in the settings. 
+    * Note: this is optional. You can reference CSS from any directory or rely on sitewide styling to get applied to your block on production. For purposes of this tutorial
+
+Your blocks directory shoud look like this: 
 
-function my_acf_init()
-{
-    // Bail out if function doesn’t exist.
-    if (!function_exists('acf_register_block')) {
-        return;
-    }
-
-    // Register a new block.
-    acf_register_block([
-        'name' => 'example_block',
-        'title' => __('Example Block', 'your-text-domain'),
-        'description' => __('A custom example block.', 'your-text-domain'),
-        'render_callback' => 'my_acf_block_render_callback',
-        'category' => 'formatting',
-        'icon' => 'admin-comments',
-        'keywords' => ['example'],
-    ]);
-}
+```
+.
+|--...
+|-- wp-content
+|   |-- themes 
+|       |-- your-theme # your theme directory
+|           |-- blocks # your blocks directory
+|               |-- my-block # your block
+|                   |-- block.json # your block settings 
+|                   |-- my-block.css # styles for your block
 ```
 
-Next, you you have to create your `render_callback()` function:
+#### Write the Block Settings file: block.json
 
-```php
-/**
- *  This is the callback that displays the block.
- *
- * @param   array  $block      The block settings and attributes.
- * @param   string $content    The block content (emtpy string).
- * @param   bool   $is_preview True during AJAX preview.
- */
-function my_acf_block_render_callback($block, $content = '', $is_preview = false)
+[Wordpress has a full example](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/) of the values you can add to your block.json. For our example, we'll follow [ACF's example](https://www.advancedcustomfields.com/resources/acf-blocks-key-concepts/#acf-blocks-and-blockjson) with the minimal fields plus the acf property. We will use renderCallback instead of renderTemplate so we can set a formula to render our blocks instead of rednering them one by one in our code. 
```

Comment:
```suggestion
[WordPress has a full example](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/) of the values you can add to your block.json. For our example, we'll follow [ACF's example](https://www.advancedcustomfields.com/resources/acf-blocks-key-concepts/#acf-blocks-and-blockjson) with the minimal settings plus the specific ACF property. We will use renderCallback instead of renderTemplate so we can set a function callback to render all our blocks instead of creating a function for each individual block.
```